Khurram_Mohammad-Abdullah_hw2

mohammadabdullahkhurram opened the pull request:
Homework 2 for Khurram, Mohammad Abdullah. Used another branch labelled "Khurram_Mohammad-Abdullah_hw2"

kochaika reviewed (changes requested):
**You are deleting other files!**

mohammadabdullahkhurram commented:
> **You are deleting other files!**

Dear Professor, 

Hope this finds you well. The same issue was there during submitting my first homework PR. Basically, I removed the other folders that got attached to my repository when I copied it the first time. Those folders were not mine so I cleaned them up before submitting my homework.  Since I submitted HW2 before my first HW was merged I branched it and so the deleting of folders from first HW also got transferred in the branch; not sure how. But all files of my homework have been submitted on time and with completion. Please let me know if there is anything else or anything I misunderstood. Thank you!

kochaika commented:
You should fix it. Not to explain how you made this mistake.
